config/routes.rb

```diff
@@ -37,4 +37,5 @@
   end
 
   get 'error_500', to: 'error#error_500'
+  get 'error_nodb', to: 'error#error_nodb'
```

Comment:
Should this just be `503` ?

Reply:
I don't think so. The intention was to test what happens when we get a db error, and yes it should be a 503, but that's not what we're explicitly triggering.

Reply:
`error_nosql_503`?